packages/back-end/src/services/stripe.ts

```diff
@@ -122,3 +122,25 @@ export async function getStripeCustomerId(org: OrganizationInterface) {
 
   return id;
 }
+
+export async function startFreeTrial(
```

Comment:
do we want to keep this? it's an unused function to start a free non-billable trial programmatically without requiring a Stripe redirect or filling out a payment form.